examples/app.py

```diff
@@ -31,10 +31,11 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
         pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
+        pip install -r requirements-devel.txt
```

Comment:
The only thing that `requirements-devel.txt` installs is DoJSON, why not put it here directly?